app/models/batch_context.rb

```diff
@@ -92,7 +92,7 @@ def object_manifest_rows
 
   # load the manifest.csv file and verify there is at least one object and the correct header is present
   def load_object_manifest
-    raise 'manifest file missing or empty' if !File.exist?(object_manifest_path) || File.zero?(object_manifest_path)
+    raise 'manifest file missing or empty' if !File.exist?(object_manifest_path) || File.empty?(object_manifest_path)
```

Comment:
Rubocop asked for this and it seems a reasonable change to me.

Reply:
I suspect it was once "empty?" and rubocop requested it be "zero?" at one point.  (eye roll)